src/java/picard/sam/markduplicates/MarkDuplicates.java

```diff
@@ -79,7 +80,12 @@
                     "READ_PAIR_OPTICAL_DUPLICATES.\n" +
                     "\n" +
                     "Usage example: java -jar picard.jar MarkDuplicates I=input.bam \\\n" +
-                    "                 O=marked_duplicates.bam M=marked_dup_metrics.txt\n";
+                    "                 O=marked_duplicates.bam M=marked_dup_metrics.txt\n" +
+                    "\n" +
+                    "The program can take either coordinate-sorted or coordinate-sorted input, however the behaviour is slightly different." +
```

Comment:
I make this (specific) spelling "error" all the time..I guess I learnt :gb: back in Israel...